benefits/eligibility/templates/eligibility/start--mst-agency-card.html

```diff
@@ -6,19 +6,15 @@
 {% endblock page-title %}
 
 {% block headline %}
-  <div class="col-lg-8">
+  <div class="col-lg-6">
```

Comment:
We probably could do something on the `start.html` template so that we don't have to repeat
```
<div class="col-lg-6">
   <h1>
...
   </h1>
</div>
```
on every one of these templates.

Something like
```
{% block headline %}
<div class="col-lg-6">
   <h1>
   {% block headline-text %}
   {% endblock headline-text %}
   </h1>
</div>
{% endblock headline %}
```
and then the `start--<agency>--<flow>.html` templates just override the `headline-text` block.

I had the thought so wrote it down, but I don't feel strongly about doing it or not. It's really not that much duplicate code, but then again we did have to remember to update the `8` to `6` in multiple places. :woman_shrugging:

Reply:
I am thinking about doing this for all templates in `base.html` in #2540, but I like the idea of starting that effort here for all the `start`-agency-flow templates!

Reply:
Done! https://github.com/cal-itp/benefits/pull/2583/commits/cf11c044ffc0760e20242517547e33715c3a18d0 Really like how the `start-agency-flow` templates are looking now, very clean and stream-lined.